server/service/AirspaceService.js

```diff
@@ -19,16 +19,8 @@ const service = {
     });
   },
 
-  /**
-   * class='W' will not be retrieved
-   */
   getAllRelevant: async () => {
     const result = await Airspace.findAll({
-      where: {
-        class: {
-          [Op.notIn]: ["W"],
```

Comment:
Aber dann sehen wir doch den W Luftraum wenn man da drauf flicht und nicht den eigentlichen. Das hatte ja irgendwie schon nen Sinn, das wir das so gemacht haben damals. Der W liegt ja immer komplett innerhalb eines anderen LR

Reply:
Wie in folgendem Screenshot, sieht man anhand der "dunkleren" Überlagerung das da eigentlich noch mehr ist.
Aber ich kann das Pop-Up noch etwas tunen, so dass beide Airspaces erwähnt werden.

Reply:
@JEK58 

So besser?
![grafik](https://github.com/XCCup/xccup.net/assets/42967368/ceb8e2c4-1244-4bf3-9281-9076d4ca2295)

Wäre es nur umständlich möglich die Daten des darunter liegenden Luftraums im selben Pop-Up anzuzeigen.

Reply:
> @JEK58
> 
> So besser? ![grafik](https://user-images.githubusercontent.com/42967368/241798946-ceb8e2c4-1244-4bf3-9281-9076d4ca2295.png)
> 
> Wäre es nur umständlich möglich die Daten des darunter liegenden Luftraums im selben Pop-Up anzuzeigen.

Ne eigentlich nicht, man sieht ja den eigentlichen Airspace nicht und die meisten Leute raffen das mit den Segelflugsektoren ja offensichtlich eh nicht.
Ich würde die einfach garnicht anzeigen und ignorieren. Die liegen eh immer innerhalb eines LR auf den wir prüfen. Für den seltenen Fall, dass jemand einen Segelflugsektor nutzt wird er das plausibel erklären können. Alles andere verwirrt glaube ich nur oder verleitet zu dummen Aktionen wegen Halbwissen.

Reply:
Und das ganze farblich anders darzustellen?

flyxc.app macht das z.B. so:
![grafik](https://github.com/XCCup/xccup.net/assets/42967368/5af7eb8a-d8c6-459b-a7ee-e0fb348bdaca)

Reply:
Ja könnte man machen, aber mit gelb signalisiert man irgendwie auch, dass es nicht so schlimm wäre da einzufliegen. Ich würde es tatsächlich lassen wie es ist. 99% der Piloten sollten und dürfen da eh nicht rein;)